testing/scenarios/vue/main.vue

```diff
@@ -0,0 +1,16 @@
+// oneline comment, malformed TODO
+// TODO 1: oneline comment wellformed
+
+<!-- TODO 3: wellformed HTML multiline entry -->
+<!-- TODO: missing number -->
+
+/* TODO 1: wellformed CS/JS multiline entry */
+/* TODO 2: wellformed CS/JS multiline entry, BUT issue closed */
+/*
+wellformed CS/JS multline entry
+*/
+
+<!--
+TODO 4: wellformed HTML multiline entry
+-->
+
```

Comment:
Could you also add an example of a string with a comment inside & verify it's not matched?

e.g.
 * "this is a // valid comment in a string"
 * "this is a /* valid multiline comment in a string */"
 * etc for the rest of the comment types

Reply:
Additionally, could you add examples for:
 * wellformed multiline HTML comment \w issue closed
 * wellformed single-lien comment \w issue closed
 * malformed multiline CSS comment